Cargo.toml

```diff
@@ -11,14 +11,20 @@ members = [
   "lucet-runtime/lucet-runtime-tests",
   "lucet-spectest",
   "lucet-wasi",
-  "lucet-wasi/generate",
   "lucet-wasi-fuzz",
   "lucet-wasi-sdk",
   "lucet-wiggle",
   "lucet-wiggle/macro",
   "lucet-wiggle/generate",
   "lucetc",
   "fuzz",
+  "wasmtime-new-api/crates/wiggle",
+  "wasmtime-new-api/crates/wiggle/borrow",
+  "wasmtime-new-api/crates/wiggle/generate",
+  "wasmtime-new-api/crates/wiggle/macro",
+  "wasmtime-new-api/crates/wasi-common",
+  "wasmtime-new-api/crates/wasi-common/cap-std-sync",
+  "wasmtime-new-api/crates/wasi-common/tokio",
```

Comment:
Could these be brought in via submodule instead of directly vendored? It would make potential security updates etc easier to apply.

Also, has this vendored code been reviewed in its home context?

Reply:
It potentially could be, but I intend for this to be short-lived enough that it doesn't matter. I am expecting it to land the main repo in a few weeks, so I didn't think it was worthwhile to add a submodule and delete it.

Yes, I have reviewed it in its home context, but not formally signed off yet.

Reply:
Gotcha. Can we leave some breadcrumbs to help apply any fixes that might be made before the formal sign-off? Like a marker of what git revision it's based off, and which branch/PR to watch over in `wasmtime`?

Reply:
Good idea, I've added this now.